### PR TITLE
Enhancement: Add validation for trading fees, blockchains, transfers

### PR DESCRIPTION
**Pull request by chumaknadya**

### CODE REVIEW

**Review by mod — CHANGES_REQUESTED** [review]
there is no point of rejecting things like "Any" or "Any   " or "   ANY   "

you must clean the string and lowercase it
No need to reject at API level, especially it will break since current tower change case by mistake

File: app/models/blockchain.rb [hunk 1]
```diff
@@ -14,6 +14,8 @@ class Blockchain < ApplicationRecord
   validates :server, url: { allow_blank: true }
   validates :client, inclusion: { in: -> (_) { clients.map(&:to_s) } }
 
+  before_create { self.key = self.key.downcase }
```
Comment (mnaichuk) [inline app/models/blockchain.rb]: please put strip here too

File: app/models/trading_fee.rb [hunk 2]
```diff
@@ -76,6 +76,8 @@ class TradingFee < ApplicationRecord
 
   # == Callbacks ============================================================
 
+  before_create { self.group = self.group.downcase }
```
Comment (mnaichuk) [inline app/models/trading_fee.rb]: please put strip here too

File: app/models/transfer.rb [hunk 3]
```diff
@@ -37,6 +37,7 @@ class Transfer < ApplicationRecord
 
   # == Callbacks ============================================================
 
+  before_create { self.key = self.key.downcase }
```
Comment (mnaichuk) [inline app/models/transfer.rb]: please put strip here too

**Review by mnaichuk — CHANGES_REQUESTED** [review]
Also, put validation for setting user group

**Review by mnaichuk — APPROVED** [review]

**Pull request merged**